README.md

```diff
@@ -6,20 +6,22 @@ Please note that `teleporter-token-bridge` is still under active development and
+
+Hub contract instances specify either an ERC20 token or the native token to be bridged, and allow for transferring the token to any registered spoke instances. The token representation on the spoke chain can also either be an ERC20 or native token, allowing users to have any combination of ERC20 and native tokens between hub and spoke chains:
 
 - `ERC20` -> `ERC20`
 - `ERC20` -> `Native`
 - `Native` -> `ERC20`
 - `Native` -> `Native`
 
-The destination tokens are designed to by default have compatibility with the token bridges on the source chain, and allow any custom logic to be implemented in addition. For example, developers can inherit and extend the destination ERC20 token contract to add additional functionality, such as a custom minting, burning, or transfer logic.
+The spoke tokens are designed to by default have compatibility with the token bridge on the hub chain, and allow custom logic to be implemented in addition. For example, developers can inherit and extend the `ERC20TokenSpoke` contract to add additional functionality, such as a custom minting, burning, or transfer logic.
 
-The token bridge also supports "multi-hop" transfers, where tokens can be transferred between destination chains. The multi-hop transfer first transfers the token from the origin destination chain to the home chain, where token balances are updated, and then triggers a second transfer to the final destination chain.
+The token bridge also supports "multi-hop" transfers, where tokens can be transferred between spoke chains. The multi-hop transfer first transfers the token from the origin spoke chain to the hub chain, where token balances are updated, and then triggers a second transfer to the final spoke chain.
 
-In addition to supporting basic token transfers, the token bridge contracts offer a `sendAndCall` interface for atomically bridging tokens and using them to interact with a smart contract on the destination chain. If the call to the recipient smart contract fails, the bridged tokens are sent to a fallback recipient address. The `sendAndCall` interfaces enables the direct use of bridged tokens in dApps on other chains, such as performing swaps, using the tokens to pay for fees when invoking services, etc.
+In addition to supporting basic token transfers, the token bridge contracts offer a `sendAndCall` interface for bridging tokens and using them in a smart contract interaction all within a single Teleporter message. If the call to the recipient smart contract fails, the bridged tokens are sent to a fallback recipient address. The `sendAndCall` interface enables the direct use of bridged tokens in dApps on other chains, such as performing swaps, using the tokens to pay for fees when invoking services, etc.
```

Comment:
Nit: "If the call to the recipient smart contract fails, the bridged tokens are sent to a fallback recipient address": Should we say on which chain the bridged tokens will be sent to here?